benchmark/acl/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 3.2.2)
```

Comment:
why 3.2.2?

Reply:
Because we have that in the other benchmark cmake files as well. I don't know which version I should require so unless a cmake pro (e.g @mmha) suggests something else, I will use the same version than in the other similar files for consistency.

Reply:
3.4.2

Reply:
Ok, fixed for all the benchmarks